council/gitflow.md

```diff
@@ -0,0 +1,243 @@
+git commit -m "Fix something. Bumped version number to 1.2.1"
+
+# push new branch and open a pull request pointing to master
+$ git push origin hotfix/1.2.1
+```
+
+After merge with *master* you need to update the *develop* branch with the hotfix too
+to avoid conflicts and loose code. 
+
+```bash
+$ git checkout develop
+Switched to branch 'develop'
+
+$ git merge --no-ff hotfix/1.2.1
+Merge made by recursive.
+(Summary of changes)
+```
+
+Hotfix branches should always branch off from `master` branch and must merge back into
+`master` too. Also this should be merged in `develop` too. The branch naming convention 
+for hotfix branches should be anything like `hotfix/something`, where something can be 
+anything except master, develop, release-*, or hotfix-*.
+
+## Release branches
+
+Release branches support preparation of a new production release. They allow for 
+last-minute dotting of i’s and crossing t’s. Furthermore, they allow for minor bug fixes 
+and preparing meta-data for a release (version number, build dates, etc.). By doing all 
+of this work on a release branch, the develop branch is cleared to receive features for 
+the next big release.
+
+The key moment to branch off a new release branch from develop is when develop (almost) 
+reflects the desired state of the new release. At least all features that are targeted 
+for the release-to-be-built must be merged in to develop at this point in time. All 
+features targeted at future releases may not—they must wait until after the release 
+branch is branched off.
+
+It is exactly at the start of a release branch that the upcoming release gets assigned 
+a version number—not any earlier. Up until that moment, the develop branch reflected 
+changes for the “next release”, but it is unclear whether that “next release” will 
+eventually become 0.3 or 1.0, until the release branch is started. That decision is 
+made on the start of the release branch and is carried out by the project’s rules 
+on version number bumping.
+
+![Release model](img/release-branch.png "Release model")
+
+Release branches are created from the develop branch. For example, say version 
+1.1.5 is the current production release and we have a big release coming up. The 
+state of develop is ready for the “next release” and we have decided that this will 
+become version 1.2 (rather than 1.1.6 or 2.0). So we branch off and give the release 
+branch a name reflecting the new version number:
+
+```bash
+# certify you are at the develop branch
+$ git status                       
+On branch develop
+
+# update branch
+$ git fetch --all --prune
+$ git pull origin develop
+
+# create release branch
+$ git checkout -b release/name-of-release
+Switched to a new branch "release/name-of-release"
+```
+
+After that open a PR pointing to `master` to put the release branch inside the 
+approval flow. After approved the responsible for open the PR must merge into `master`,
+delete the release branch and **tag new version**.
```

Comment:
At the moment, `master` is protected in a way that only some members can add commits. So I'd rephrase this a bit because it gives the sensation that "anyone can do it". Since the repo is public this protection is a must.